stable/velero/templates/restic-daemonset.yaml

```diff
@@ -96,4 +96,8 @@ spec:
           resources:
             {{- toYaml . | nindent 12 }}
           {{- end }}
+      {{- with .Values.restic.tolerations }}
+      tolerations:
+        {{- toYaml . | nindent 8 }}
```

Comment:
I believe this should be inside the tolerations in case anyone needs to add tolerations too. This property is not only defines for restic. I suggest to allow defining tolerations as a generic property